fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/digitalcash/TxOut.java

```diff
@@ -0,0 +1,49 @@
+package com.github.dedis.popstellar.model.network.method.message.data.digitalcash;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Objects;
+
+// Object representing an output of this transaction
+public class TxOut {
+  @SerializedName("Value")
+  private final int value; // the value of the output transaction, expressed in miniLAOs
```

Comment:
```suggestion
  private final long value; // the value of the output transaction, expressed in miniLAOs
```
optional, but given the planned range of transaction amounts this could turn out to be handy. (also hit the constructor and the accessor)